game/hud/src/widgets/Welcome/index.scss

```diff
@@ -0,0 +1,38 @@
+/**
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+$cu-styles-root: '../../../interface-lib/camelot-unchained/styles/';
+@import "camelot-unchained/lib/index.scss";
+
+.cu-welcome {
```

Comment:
Use OOCSS naming as per [style guide](https://github.com/airbnb/css#oocss-and-bem).

This would be based on your React component names.  So.  `.cu-welcome` would be instead `Welcome` and sub elements defined within the welcome component would be `Welcome__name`.  So that button bar should be `Welcome__btnBar` and the dismiss button would be `Welcome__btnBar__dismiss`.

This should help explain it: http://getbem.com/naming/